src/AddressRegistry.Api.BackOffice.Handlers.Sqs/Handlers/SqsAddressApproveHandler.cs

```diff
@@ -0,0 +1,46 @@
+namespace AddressRegistry.Api.BackOffice.Handlers.Sqs.Handlers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Abstractions;
+    using Be.Vlaanderen.Basisregisters.MessageHandling.AwsSqs.Simple;
+    using Microsoft.Extensions.Logging;
+    using Requests;
+    using TicketingService.Abstractions;
+
+    public class SqsAddressApproveHandler : SqsHandler<SqsAddressApproveRequest>
+    {
+        public const string Action = "ApproveStreetName";
+        private readonly BackOfficeContext _backOfficeContext;
+
+        public SqsAddressApproveHandler(
+            ISqsQueue sqsQueue,
+            ITicketing ticketing,
+            ITicketingUrl ticketingUrl,
+            BackOfficeContext backOfficeContext)
+            : base(sqsQueue, ticketing, ticketingUrl)
+        {
+            _backOfficeContext = backOfficeContext;
+        }
+
+        protected override string? WithAggregateId(SqsAddressApproveRequest request)
+        {
+            var relation = _backOfficeContext
+                .AddressPersistentIdStreetNamePersistentIds
+                .Single(x => x.AddressPersistentLocalId == request.Request.PersistentLocalId);
```

Comment:
Can you change Single() into Find()? and below change it to to relation**?.**StreetNamePersistentLocalId.Tostring();?